lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -72,6 +72,7 @@
                 new FieldsView.DropdownFieldView({
                     model: accountSettingsModel,
                     screenReaderTitle: gettext('Country'),
+                    screenReaderLabel: gettext('Add Country'),
```

Comment:
@cptvitamin @ahsan-ul-haq Should the label continue to say "Add Country" even after the user has set some value? Wouldn't something like 'Specify Country' make more sense?